guides/development-workflow.md

```diff
@@ -448,6 +514,9 @@ have chosen the following existing style guides for our work:
   [Sass Guidelines - Syntax and formatting](https://sass-guidelin.es/#syntax--formatting)
 - JavaScript - [Standard](https://standardjs.com/)
 
+When joining an existing code base we suggest following any existing code style
+rather than switch over to ours.
```

Comment:
If we join an existing codebase without style linting, we should probably think about adding some rather than living with it. This might involve investigating what style the existing code is closest to.